inject-groovy/src/test/groovy/io/micronaut/ast/groovy/visitor/GroovyReconstructionSpec.groovy

```diff
@@ -333,12 +335,12 @@ class Wrapper<U> {
         fieldType                               | expectedType
         'String'                                | 'String'
         'List<String>'                          | 'List<String>'
-        'List<T>'                               | 'List<U>'
```

Comment:
Those tests were modified in the previous PR. Now it's back to how it was before.